Resources/Private/JavaScript/src/hooks/useSelectTag.ts

```diff
@@ -0,0 +1,19 @@
+import { useSetRecoilState } from 'recoil';
+import { Tag } from '../interfaces';
+import { selectedAssetCollectionIdState, selectedAssetIdState, selectedTagState } from '../state';
+import selectedInspectorViewState from '../state/selectedInspectorViewState';
+
+const useSelectTag = () => {
+    const setSelectedAssetCollectionId = useSetRecoilState(selectedAssetCollectionIdState);
+    const setSelectedTag = useSetRecoilState(selectedTagState);
+    const setSelectedAssetId = useSetRecoilState(selectedAssetIdState);
+    const setSelectedInspectorView = useSetRecoilState(selectedInspectorViewState);
+
+    return (tag: Tag, assetCollection = null) => {
```

Comment:
`assetCollection` should have a type specified as it's considered `any` right now